<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySerialPort</class>
 <widget class="QWidget" name="MySerialPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>射频通信</string>
   </property>
   <widget class="QComboBox" name="rfSerialPort_ComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="rfBaud_ComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>56</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rfOpen_Btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rfFlush_Btn">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>110</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>230</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>水声通信</string>
   </property>
   <widget class="QPushButton" name="soundOpen_Btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="soundFlush_Btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QComboBox" name="soundBaud_Combox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="soundSerialPort_Combox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>230</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>卫星通信</string>
   </property>
   <widget class="QPushButton" name="sateOpen_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sateFlush_Btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sateBaud_Combox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="sateSerialPort_Combox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>UDPHost</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="hostIP_Combox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>192.168.1.141</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192.168.1.10</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="hostPort_Combox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>8080</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="udpConnect_Btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disConnect_Btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>UDPSlave</string>
   </property>
   <widget class="QComboBox" name="slaveIP_Combox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>192.168.1.117</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192.168.1.12</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="slavePort_Combox">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>90</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8080</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>IP </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_Btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>通信方式</string>
   </property>
   <widget class="QCheckBox" name="rf_CheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>27</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>射频</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sound_CheckBox">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>27</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>水声</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sate_CheckBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>26</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>卫星</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="net_CheckBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>26</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>网络</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>自动刷新</string>
   </property>
   <widget class="QCheckBox" name="flushSpeed_CheckBox">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>27</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>速度</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="flushSenser_CheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>26</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>传感器状态</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string> 调试</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="debug_Btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
